docs/index-server.md

```diff
@@ -180,6 +179,12 @@ that will authenticate the user to access the index server and no other service.
 | Name     |      Default Value    |  Description |
 |:----------:|:-------------:|:------:       |
 | ipc.client.rpc-timeout.ms |  NA | Set the above property to some appropriate value based on your estimated query time. The best option is to set this to the same value as spark.network.timeout. |
+| hadoop.security.authorization |  false | property to enable the Hadoop Security Enable.(Need only Server Side). |
```

Comment:
Change it from `property to enable the Hadoop Security Enable.(Need only Server Side). `  to `Property to enable the hadoop security which is required only on the server side.`

Reply:
OK